src/pyosmeta/cli/process_reviews.py

```diff
@@ -48,12 +48,10 @@ def main():
         print("-" * 20)
 
     # Update gh metrics via api for all packages
-    print("Getting GitHub metrics for all packages...")
-    repo_endpoints = process_review.get_repo_endpoints(accepted_reviews)
-    all_reviews = process_review.get_gh_metrics(
-        repo_endpoints, accepted_reviews
-    )
-
+    # BUG : contrib count isn't correct - great tables has some and is returning 0
```

Comment:
This isn't a bug, the endpoint isn't available via graphql.

Reply:
I've updated the PR to use rest just for that metric. it reduces the api calls by one per package vs two now.